error_test.go

```diff
@@ -23,9 +23,7 @@ func TestErrorResponse(t *testing.T) {
 	defer ts.Close()
 
 	SetBackend("api", BackendConfiguration{
-		APIBackend,
-		ts.URL,
-		&http.Client{},
+		Type: APIBackend, URL: ts.URL, HTTPClient: &http.Client{},
 	})
 
 	err := GetBackend(APIBackend).Call("GET", "/v1/account", "sk_test_badKey", nil, nil, nil)
```

Comment:
Not something to do in this PR, but probably switching these `"GET"`, and other http method calls to the official `http.MethodGet` types and all that might be less error prone, -- not that this is currently the source of any issues.